src/sinks/prometheus/exporter.rs

```diff
@@ -255,6 +251,10 @@ impl PrometheusExporter {
                         )
                     });
 
+                    emit!(PrometheusServerRequestComplete {
+                        code: response.status().as_u16(),
```

Comment:
Could this be stored as the actual type of the status instead of converting to `u16`?

Reply:
Sure. Updated.